adafruit_funhouse/peripherals.py

```diff
@@ -27,6 +27,7 @@
 
 """
 
+from typing import Optional
```

Comment:
I think this needs to be a `try/except` block the same way the typing imports in __init__.py are done

Reply:
Damn, I think VS Code snuck that in there when I typed Optional, good catch!

Reply:
Missed this one.